Fix build failure with libxml2 >= 2.12.0:

error: use of undeclared identifier 'xmlFree'

https://gitlab.com/inkscape/inkscape/-/issues/4753
https://gitlab.com/inkscape/inkscape/-/commit/694d8ae43d06efff21adebf377ce614d660b24cd
--- src/object/uri.h.orig
+++ src/object/uri.h
@@ -13,6 +13,7 @@
 #define INKSCAPE_URI_H
 
 #include <libxml/uri.h>
+#include <libxml/xmlmemory.h>
 #include <memory>
 #include <string>
 
